core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -40,7 +39,7 @@ public override void addPassword(string serviceName, string user, string passwor
             var hostUrl = $"{serviceName} - {user}";
             if (!WinCredentialManager.SaveCredentials(hostUrl, new NetworkCredential(user, password)))
             {
-                throw new LocalAccessDeniedException($"Could not save credentials for \"{hostUrl}\" to Windows Credential Manager.");
+                logger.error($"Could not save credentials for \"{hostUrl}\" to Windows Credential Manager.");
```

Comment:
Why no longer throwing failure?

Reply:
Is there a difference in throwing? Will be written into error log regardless, and prevents unecessary stack rewinds and exception handling.